src/AbstractPluginManager.php

```diff
@@ -49,21 +48,11 @@ abstract class AbstractPluginManager extends ServiceManager implements PluginMan
      * factories; for $config, {@see \Laminas\ServiceManager\ServiceManager::configure()}
      * for details on its accepted structure.
      *
-     * @param null|ConfigInterface|ContainerInterface|PsrContainerInterface $configInstanceOrParentLocator
+     * @param null|ConfigInterface $configInstanceOrParentLocator
```

Comment:
We should keep here `ContainerInterface` (`Interop`) as it is `Psr` through the alias.